Header header
string child_frame_id
float64 desired_velocity_x
float64 desired_velocity_z
float64 initial_pose_x
float64 initial_pose_y
float64 initial_twist_z
float64 final_pose_x
float64 final_pose_y
float64 final_twist_z
float64 stop_distance
float64 stop_time
float64 dist_error
